picard/ui/mainwindow.py

```diff
@@ -799,8 +799,12 @@ def play_file(self):
 
     def open_folder(self):
         files = self.tagger.get_files_from_objects(self.selected_objects)
+        opened = set()
         for file in files:
-            QtGui.QDesktopServices.openUrl(self._openUrl(os.path.dirname(file.filename)))
+            url = self._openUrl(os.path.dirname(file.filename))
+            if not url in opened:
+                QtGui.QDesktopServices.openUrl(url)
+                opened.add(url)
```

Comment:
@Sophist-UK : i thought about it yesterday, and yes, i think now it is a bad move, it may cause more issues than it solves. I will drop the `os.path.realpath` part.